src/components/ImageSlider.tsx

```diff
@@ -8,34 +8,25 @@ type ImageSliderProps = {
 
 const ImageSlider = ({ urls }: ImageSliderProps) => {
```

Comment:
타입스크립트에서 함수에 타입을 지정할 때는 다음 방법이 좋아요

e.g.

```ts
// ❌
const sum = (a: number, b: number): number => a + b

// ✅
type SumHandler = (a: number, b: number) => number
const sum: SumHandler = (a, b) => a + b
```

Reply:
이 방법대로라면 타입스크립트에서 컴포넌트 타입을 지정할 때는 다음 방법이 좋아요

```suggestion
const ImageSlider: React.FC<ImageSliderProps> = ({ urls }) => {
```